playbooks/bootstrap-node.yml

```
@@ -0,0 +1,139 @@
+# ==============================================================================
+# Node Bootstrapper Playbook
+#
+# Logic: Dual-Play transition for secure identity bootstrapping across clusters.
+# Play 1: Rootful Bootstrap (Target Initial User -> dsom-admin creation UID/GID 2001)
+# Play 2: System Preparation (Target dsom-admin -> Full Configuration)
+# ==============================================================================
+
+# --- [ PLAY 1: IDENTITY BOOTSTRAP (ROOTFUL) ] ---
+- name: "Bootstrap | Node Identity Setup"
+  hosts: "{{ target_host | default('all') }}"
+  tags: [ bootstrap, setup ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)